.github/workflows/test.yml

```diff
@@ -135,24 +133,40 @@ jobs:
           echo "LIBRARY_PATH=${pwd_slash}/libgcc_mock" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
 
           # Add R bin path to PATH
-          echo "$(Rscript.exe -e 'cat(normalizePath(R.home()))')\bin\x64"  | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append ;
+          echo "$(Rscript.exe --vanilla -e 'cat(normalizePath(R.home()))')\bin\x64"  | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append ;
 
           # Add Rtools' GCC to PATH
-          if ($env:RTOOLS_VERSION -eq '44') {
-            $mingw_root = "C:\rtools44\x86_64-w64-mingw32.static.posix"
-          } elseif ($env:RTOOLS_VERSION -eq '43') {
-            $mingw_root = "C:\rtools43\x86_64-w64-mingw32.static.posix"
-          } elseif ($env:RTOOLS_VERSION -eq '42') {
-            $mingw_root = "C:\rtools42\x86_64-w64-mingw32.static.posix"
+          # Source: https://github.com/r-lib/actions/blob/b7e68d63e51bdf225997973e2add36d551f60f02/setup-r/lib/installer.js#L471
+          $directories = @(
+            "C:\rtools44-aarch64\aarch64-w64-mingw32.static.posix",
+            "C:\rtools44\x86_64-w64-mingw32.static.posix",
+            "C:\rtools43\x86_64-w64-mingw32.static.posix",
+            "C:\rtools42\x86_64-w64-mingw32.static.posix",
```

Comment:
How about maybe scanning for `rtools??` folders? Otherwise, you will have to update this if there will be `rtools45` and so on.

Reply:
We already do that. And the ARM support will be tricky.
I also don't know how to write this with little failure points.